### PR TITLE
Minor doc fixes

### LINKED ISSUE #182: Fix named anchors in the documentation
Not all `<href>` point to named anchors that exist, and on some pages there exist non-unique named anchors. Fixing this should make it easier for people to link to a specific point in the documentation.

### PR DESCRIPTION
**Pull request by devknoll**
Partially fixes #182 and another small issue inconsistency I found. This doesn't fix nested anchors yet.

### CODE REVIEW

File: docs/Interfaces-NetworkLayer.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Custom network layers that must conform to the `RelayNetworkLayer` interface.
 
 <ul class="apiIndex">
   <li>
-    <a href="#sendmutations">
+    <a href="#sendmutation">
       <pre>sendMutations(mutationRequest)</pre>
```
Comment (fson) [inline docs/Interfaces-NetworkLayer.md]: The text of the link is still wrong?

Comment (devknoll) [inline docs/Interfaces-NetworkLayer.md]: Fixed, thanks!

**Comment by devknoll** [comment]
@steveluscher For the separate problem of duplicate/nested anchors, I'm currently thinking of just modifying https://github.com/facebook/relay/blob/master/website/core/Header.js to keep the slug in the context and nesting them on render.

Sound good?

**Comment by steveluscher** [comment]
:+1:

**Comment by devknoll** [comment]
Went a slightly different direction with this. I noticed that the anchor links break down at around H4, everything above that is unique per page. So this will start generating nested anchors at H4. The benefit of this is that all of the existing anchors will continue to work.

WTB tests or anchor testing software :wink: 

cc @steveluscher

**Pull request merged**